src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+abstract class Figure {
+    protected Color color;
+
+    public void setColor(Color color) {
+        this.color = color;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public abstract double area();

```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. It seems that `area()` and `draw()` methods should not be in the same abstract class/interface.